item-spec/item-spec.md

```diff
@@ -251,6 +228,13 @@ in the file).
 For examples of fields that this construct is recommended for, see the [section of STAC Best Practices](../best-practices.md#common-use-cases-of-additional-fields-for-assets)
 that talks about common use cases of additional fields for assets.
 
+## Media Type for STAC Item
+
+A STAC Item is a GeoJSON file ([RFC 7946](https://tools.ietf.org/html/rfc7946)), and thus should use the 
+`[application/geo+json](https://tools.ietf.org/html/rfc7946#section-12)` as the [Media Type](https://en.wikipedia.org/wiki/Media_type) 
+(previously known as the MIME Type). STAC Items are recommended to add the parameter `profile` with the value `stac-item` to
```

Comment:
This is the same issue as with GeoTiff. You can't add parameters, I think it should just be the GeoJSON media type. Also, I think we should say that `application/json` can also occur in implementations. I don't think everybody implemented the GeoJSON media type.

Reply:
Ah, meant to remove the 'profile' bit. Good idea on saying application/json is ok too.